### PR TITLE
Amb 816 add sandbox

### PR DESCRIPTION
**Pull request by artronics**
## Summary
* Routine Change
* :exclamation: Breaking Change
* :robot: Operational or Infrastructure Change
* :sparkles: New Feature
* :warning: Potential issues that might be caused by this change

Add any other relevant notes or explanations here. **Remove this line if you have nothing to add.**

## Reviews Required
* [x] Dev
* [ ] Test
* [ ] Tech Author
* [ ] Product Owner

## Review Checklist
:information_source: This section is to be filled in by the **reviewer**.

* [ ] I have reviewed the changes in this PR and they fill all or part of the acceptance criteria of the ticket, and the code is in a mergeable state.
* [ ] If there were infrastructure, operational, or build changes, I have made sure there is sufficient evidence that the changes will work.
* [ ] I have ensured the changelog has been updated by the submitter, if necessary.

### CODE REVIEW

File: azure/templates/run-tests.yml [hunk 1]
```diff
@@ -34,14 +34,25 @@ steps:
...
-        poetry run pytest --reruns 3 --reruns-delay 1 -v -m smoketest --junitxml=tests/api_tests.xml 
-      elif [[ $APIGEE_ENVIRONMENT == "int" ]]
+       run_pytest smoketest
+      elif [[ $APIGEE_ENVIRONMENT =~ .*-*sandbox ]]
       then
-        poetry run pytest --reruns 3 --reruns-delay 1 -v -m integration --junitxml=tests/api_tests.xml
+       run_pytest sandbox
       else
-        poetry run pytest --reruns 3 --reruns-delay 1 -v --junitxml=tests/api_tests.xml
+       run_pytest
```
Comment (artronics) [inline azure/templates/run-tests.yml:55]: marker `broker` can only be ran inside internal-dev

File: ecs-proxies-containers.yml [hunk 2]
```diff
@@ -1,4 +1,4 @@
 docker_containers:
   - name: sandbox
-    dockerfile: "sandbox/Dockerfile"
-    path: "sandbox"
+    dockerfile: "sandbox.Dockerfile"
```
Comment (artronics) [inline ecs-proxies-containers.yml:3]: This is wrong

File: proxies/sandbox/apiproxy/policies/ServiceCallout.CallHealthcheckEndpoint.xml [hunk 3]
```diff
@@ -7,7 +7,7 @@
     </Request>
     <Response>healthcheckResponse</Response>
     <HTTPTargetConnection>
```
Comment (artronics) [inline proxies/sandbox/apiproxy/policies/ServiceCallout.CallHealthcheckEndpoint.xml:9]: See PDS for _status service callout

File: scripts/pre-commit [hunk 4]
```diff
@@ -1,2 +1,4 @@
 #!/bin/sh
 make lint
+make copy-examples
```
Comment (artronics) [inline scripts/pre-commit:3]: remove these two lines

File: specification/examples/_status.json [hunk 5]
```diff
@@ -0,0 +1,20 @@
+{
```
Comment (artronics) [inline specification/examples/_status.json:1]: We can remove the hardcoded body. Also check mock-proxy

File: tests/example_loader.py [hunk 6]
```diff
@@ -5,7 +5,7 @@
 
 
 def load_example(path: str):
-    with open(f"{current_directory}/../specification/examples/{path}") as f:
+    with open(f"{current_directory}/../sandbox/src/routes/examples/{path}") as f:
```
Comment (artronics) [inline tests/example_loader.py:8]: move it back to specification directory

**Review by Valentinolucas — APPROVED** [review]

**Pull request merged**